addons/website_blog/static/src/interactions/website_blog.edit.js

```diff
@@ -0,0 +1,14 @@
+import { WebsiteBlog } from "./website_blog";
+import { registry } from "@web/core/registry";
+
+export const WebsiteBlogEdit = (I) =>
+    class extends I {
+        start() {
+            this.el.querySelector("#edit-in-backend")?.classList.add("d-none");
+        }
+    };
+
+registry.category("public.interactions.edit").add("website_blog.website_blog", {
+    Interaction: WebsiteBlog,
+    mixin: WebsiteBlogEdit,
+});
```

Comment:
You can simply add the `css_editable_mode_hidden` class to the button to achieve that 😉